0xProject/0x-monorepo
# Portal v2 onboarding polish

## Description

Ideally, this is the last branch before we consider onboarding ready-to-ship (at least v1).
- Tooltip always points to wallet, and so jumps around less. 
- Fixed issue where mobile onboarding did not animate correctly on Webkit based browsers (Safari + any iOS browser).
- Changed image for final onboarding step.
- Can now click outside to close onboarding. 
- Broke up some steps so that they can fit on mobile (specifically wrap ETH).
- Fixed issue where we don't scroll to wallet on mobile onboarding if onboarding starts automatically.
- Made some steps a bit more customized depending on what you've done already.
- Fixed an issue where the `skip` button was on the wrong side of the onboarding card.
- More z-index issues!
- Always start onboarding from 0.
- Go directly to relayer page on mobile after you're done with onboarding.
- Always automatically show onboarding if you're on the 0th (and therefore don't have metamask installed) step.

We decided not to skip steps for now, since ideally the majority of people going through the flow will be starting from scratch, and it' always nice to go back through the steps if you've done them already.

<img width="379" alt="screen shot 2018-06-27 at 6 28 08 pm" src="https://user-images.githubusercontent.com/3066135/42007956-e53573ee-7a37-11e8-971d-3f2656e2f4af.png">

## Testing instructions
Try it out and see if the changes above work. It has been deployed to dogfood and feels pretty good on Cipher.

## Types of changes




## Checklist:
